src/index.js

```diff
@@ -1,21 +1,17 @@
 "use strict";
 
-let WritableStream = require("./writable_stream");
+const WritableStream = require("./writable_stream");
 
-module.exports = class Renderer {
-	constructor(bundlePath) {
-		this.bundlePath = bundlePath;
-		this.reload();
-	}
+module.exports = bundle => {
+	const _render = bundle;
 
-	render(response, tag, params) {
-		// TODO: `response.status(200);`?
-		let stream = new WritableStream(response);
-		this._render(stream, tag, params);
-		response.end();
-	}
+	return (req, res, next) => {
+		res.render = (tag, params) => {
```

Comment:
I like this approach (seems much simpler than [Marko's solution](https://github.com/marko-js/marko/blob/a79621eee4eb4ff3f8efe02219b8e7fe4c5f3bde/src/express.js#L8)), but I'm wary of overriding `#render`; I'd rather we add a method of our own instead.